src/components/Home.js

```diff
@@ -0,0 +1,75 @@
+import React, { Component, Fragment } from 'react';
+import GifList from './GifList';
+import { connect } from 'react-redux';
+import { fetchTrendingGifs } from '../actions/giphyApi'
+import { toggleFavoriteGif } from '../actions/favorites';
+import NavBar from './NavBar';
+import InfiniteScroll from './InfiniteScroll';
+
+export class Home extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      search: '',
```

Comment:
This `search` term could be reused with the other view and stored in the Redux store.